From d478da9ba24e2016b76c1f54e9a75e0749d30c5a Mon Sep 17 00:00:00 2001
From: Jayanthi Annadurai <jannadurai@marvell.com>
Date: Mon, 24 May 2021 13:04:42 -0700
Subject: [PATCH 1541/1921] drivers: sdhci: Fix typo error in conditional macro

NON CN10k platform is broken due to the bug in conditional compilation.

Change-Id: Ia6acf36d4714cf3fd3b836e35b7b6a64bbb5d3d3
Signed-off-by: Jayanthi Annadurai <jannadurai@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/52528
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/mmc/host/sdhci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci.c b/drivers/mmc/host/sdhci.c
index b13c497b047b..88c7267f6e74 100644
--- a/drivers/mmc/host/sdhci.c
+++ b/drivers/mmc/host/sdhci.c
@@ -2433,7 +2433,7 @@ static int __sdhci_execute_tuning(struct sdhci_host *host, u32 opcode)
 {
 	int i;
 
-#ifndef CONFIG_MMC_SDHCI_CADENCE_WORKAROUND
+#ifdef CONFIG_MMC_SDHCI_CADENCE_WORKAROUND
 	return 0;
 #endif
 	/*
-- 
2.31.1

